docs/faq.md

```diff
@@ -20,13 +20,13 @@ import TOCInline from "@theme/TOCInline"
 
 The [Semgrep command line tool](https://github.com/returntocorp/semgrep) is open-source, licensed under [LGPL 2.1](https://tldrlegal.com/license/gnu-lesser-general-public-license-v2.1-(lgpl-2.1)) — you can use it at work, on private and proprietary code, no problem!
 
-The [Semgrep Registry](https://semgrep.dev/explore) contains rules from different contributors. Most rules, including all community-contributed rules, are under the [MIT license](https://tldrlegal.com/license/mit-license). Some of the Semgrep Registry rules written by r2c are licensed under the [Semgrep Registry license](../licensing/#semgrep-registry-license).
+The [Semgrep Registry](https://semgrep.dev/explore) contains rules from different contributors. Most rules, including all community-contributed rules, are under the [MIT license](https://tldrlegal.com/license/mit-license). Semgrep Registry rules written by r2c are licensed under the [Common Clause](https://github.com/returntocorp/semgrep-rules/blob/develop/LICENSE).
```

Comment:
Technically, LGPL 2.1 + Commons Clause

Reply:
(Fixed. I took a look at the license in the readme and was wondering why we didn't include LGPL in the header, but it's there in the bottom.)  Merging now.